pkg/runner/deb/packageconfig.go

```diff
@@ -31,5 +34,48 @@ func (p *PackageConfig) UpdatePackageGroup(g *PackageGroup) error {
 		g.Add(*p.Override...)
 	}
 
+	if p.OverrideDir != nil {
+		overrides := make([]Package, 0, len(g.packages))
+		for _, pkg := range g.packages {
+			if pkg.LocalPath != nil {
+				continue
+			}
+			if localDeb, err := findLocalDeb(pkg, p.OverrideDir); err == nil {
+				overrides = append(overrides, Package{
+					Name:      pkg.Name,
+					LocalPath: &localDeb,
+				})
+			}
```

Comment:
You're swallowing the error here, and not returning it to the user.  This means that e.g. you'll fail silently if they have 2 packages with the same root name.  That would be extremely confusing.